windows/hcsshimtypes/hcsshim.proto

```diff
@@ -16,7 +16,7 @@ message CreateOptions {
 // ProcessDetails is made of the same fields as found in hcsshim.ProcessListItem
 message ProcessDetails {
 	string image_name = 1;
-	google.protobuf.Timestamp created_at = 2;
+	google.protobuf.Timestamp created_at = 2 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
```

Comment:
how did you figure out that this needed to be added?

Reply:
and is the conversion from ProcessListItem type `time.Time` to CreatedAt type `google.protobuf.Timestamp` now done for you?

Reply:
That's what we use in the other proto files, it indeed permits us not to deal with the `google.protobuf.Timestamp` conversion as it is done by the generated code now.

One of the advantages of `gogoproto`. I think we have @stevvooe to thank for using these options originally.